src/assets/locales/en-US/translation.json

```diff
@@ -91,7 +91,7 @@
     "close": "Close",
     "addSection": "Add Section",
     "resetSections": "Reset Sections",
-    "sectionsNote": "Note: changing the default section (<1 />) will only be affected after clicking Reset Positions or Reset All."
+    "sectionsNote": "Note: changing the default section will only be affected after clicking Reset Positions or Reset All. <1 /> Default Reset - <3 /> <5 /> Default Dead - <7 /> <9 /> Default Unused - <11 />"
```

Comment:
Maybe we should rename to reset section, dead section and unused section.

Reply:
i did that initially, but wasn't sure which approach is better. i'll change it